packages/bee-q/src/components/breadcrumb-item/bq-breadcrumb-item.tsx

```diff
@@ -0,0 +1,133 @@
+import { h, Component, Prop, Element, Event, EventEmitter } from '@stencil/core';
+
+import { isDefined } from '../../shared/utils';
+
+/**
+ * @part base - The component wrapper container
+ * @part base - The breadcrumb item wrapper (`button` or `a`)
+ * @part content - The `span` tag that loads the content item
+ * @part separator - The `span` tag that loads the separator
+ */
+@Component({
+  tag: 'bq-breadcrumb-item',
+  styleUrl: './scss/bq-breadcrumb-item.scss',
+  shadow: true,
+})
+export class BqBreadcrumbItem {
+  // Own Properties
+  // ====================
+
+  // Reference to host HTML element
+  // ===================================
+
+  @Element() el!: HTMLBqBreadcrumbItemElement;
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  // Public Property API
+  // ========================
+
+  /** If true, the item is the last element inside breadcrumb */
+  @Prop() isLastItem: boolean = false;
```

Comment:
We should add an `aria-label` for the breadcrumb item that will allow consumers to place the page title in context for screen readers.

```suggestion
  /** The aria-label that corresponds to the full title of the destination page.
   * This won't be shown in the page, but it will be used by screen readers and other assistive devices */
  @Prop() ariaLabel: string;

  /** If true, the item is the last element inside breadcrumb */
  @Prop() isLastItem: boolean = false;
```

Usage example:

```html
<bq-breadcrumb-item aria-label="Hom page">
  <bq-icon name="house-line" size="16"></bq-icon>
</bq-breadcrumb-item>
```